app/src/main/java/org/flyve/mdm/agent/core/user/UserModel.java

```diff
@@ -47,6 +47,18 @@ public UserModel() {
 
     }
 
+    /**
+     * Set the object's properties to equal the arguments given
+     * @param string firstName
+     * @param string lastName
+     * @param string language
+     * @param string phone
+     * @param string mobilePhone
+     * @param string phone2
+     * @param string the administrativeNumber
+     * @param string the picture
+     * @param string the emails
```

Comment:
String or ArrayList?

Reply:
EmailsData appear in your other comments

Reply:
I'll fix it now